content/contributing/documentation.rst

```diff
@@ -312,19 +312,22 @@ with :abbr:`RST (reStructuredText)` syntax (see :doc:`documentation/rst_cheat_sh
 guidelines.
 
 .. important::
-   If your changes include the addition of a new image, for instance :file:`my_image.png`, proceed
-   as follows:
+   - If your changes include the addition of a new image, for instance :file:`my_image.png`, proceed
+     as follows:
 
-   #. Make sure that the image is in ``.png`` format.
-   #. Execute the following commands in a terminal:
+     #. Make sure that the image is in ``.png`` format.
+     #. Execute the following commands in a terminal:
 
-      .. code-block:: console
+        .. code-block:: console
 
-         $ cd path-to-the-directory-of-the-image/
-         $ pngquant my_image.png
+           $ cd path-to-the-directory-of-the-image/
```

Comment:
```suggestion
           $ cd path-to-the-directory-of-the-image/
```
folders follow the naming convention of rst files, and last time we discussed it, we said it would remain with underscores.
```suggestion
           $ cd path_to_the_directory_of_the_image/
```